src/css/custom.css

```diff
@@ -80,6 +80,7 @@
   --ifm-color-primary-lightest: #b198df;
   --docusaurus-highlighted-code-line-bg: rgba(0, 0, 0, 0.3);
   --ifm-font-color-base: rgb(218, 221, 225);
+  --ifm-font-color-base-inverse: #000000
```

Comment:
```suggestion
  --ifm-font-color-base-inverse: black;
```

Reply:
I just copied the one from the font colour base in the light theme?

Reply:
Fine by me, but add the missing `;` :)